src/user/event_service.rst

```diff
@@ -0,0 +1,141 @@
+.. _event_service:
+
+EventService
+============
+
+Workflow
+------------
+
+Here is the structure of workflow.
+
+.. image:: ../_images/work_concepts.png
+   :width: 6.5in
+   :height: 2.66667in
+
+- **Workflow**. One workfow corresponds to a scientifc objective defning intermediate steps for the user
```

Comment:
defning -> defining
Perhaps instead: objective broken into intermediate steps
(remove "for the user to accomplish" unless meant that these intermediate steps are defined by the user)